app/Models/User.php

```diff
@@ -79,6 +81,16 @@ public function account(): BelongsTo
         return $this->belongsTo(Account::class);
     }
 
+    /**
+     * Get the preferred locale of the entity.
+     *
+     * @return string
+     */
+    public function preferredLocale()
```

Comment:
Why do you use a method for a getter and not call `user->locale` directly?

Reply:
This is the implementation of the `HasLocalePreference` interface